x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_explorer/analytics_collection_explorer_callout.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiButton, EuiCallOut } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { RedirectAppLinks } from '@kbn/kibana-react-plugin/public';
+
+import { KibanaLogic } from '../../../../shared/kibana';
+import { AnalyticsCollectionDataViewLogic } from '../analytics_collection_data_view_logic';
+
+export const AnalyticsCollectionExplorerCallout: React.FC = () => {
+  const { application } = useValues(KibanaLogic);
+  const { dataView } = useValues(AnalyticsCollectionDataViewLogic);
+  const exploreLink =
+    dataView &&
+    application.getUrlForApp('discover', {
```

Comment:
Can be in a follow up PR but we should create a util function for this? I noticed that RedirectAppLinks is deprecated so we need to explore options on how to get to discover better.